Protocol.md

```diff
@@ -144,6 +158,8 @@ The receiving node MUST reject the contract if:
   - it considers `feerate_per_vb` too small for timely processing or unreasonably large.
   - `funding_pubkey` is not a valid secp256k1 pubkey in compressed format.
   - `funding_inputs` do not contribute at least `total_collateral_satoshis` plus full [fee payment](Transactions.md#fee-payment).
+  - Any `input_serial_id` is duplicated
+  - The `fund_output_serial_id` and `change_serial_id` must be set to different values
```

Comment:
nit: change to "are not set to different values" (it doesn't fit with the other sentences)